spec/index.bs

```diff
@@ -2853,29 +2929,37 @@ dictionary GPUBindGroupLayoutEntry {
 
     : <dfn>buffer</dfn>
     ::
-        When not `undefined` indicates the [=binding resource type=] for this {{GPUBindGroupLayoutEntry}}
+        When not `undefined`, indicates the [=binding resource type=] for this {{GPUBindGroupLayoutEntry}}
         is {{GPUBufferBinding}}.
 
     : <dfn>sampler</dfn>
     ::
-        When not `undefined` indicates the [=binding resource type=] for this {{GPUBindGroupLayoutEntry}}
+        When not `undefined`, indicates the [=binding resource type=] for this {{GPUBindGroupLayoutEntry}}
         is {{GPUSampler}}.
 
     : <dfn>texture</dfn>
     ::
-        When not `undefined` indicates the [=binding resource type=] for this {{GPUBindGroupLayoutEntry}}
+        When not `undefined`, indicates the [=binding resource type=] for this {{GPUBindGroupLayoutEntry}}
         is {{GPUTextureView}}.
 
     : <dfn>storageTexture</dfn>
     ::
-        When not `undefined` indicates the [=binding resource type=] for this {{GPUBindGroupLayoutEntry}}
+        When not `undefined`, indicates the [=binding resource type=] for this {{GPUBindGroupLayoutEntry}}
         is {{GPUTextureView}}.
+
+    : <dfn>externalTexture</dfn>
```

Comment:
We should also update the counting of bindings when computing whether a `GPUBindGroupLayout` is under the device's limits. I think a `GPUExternalTexture` needs to be 3 sampled views + 1 UBO?

Reply:
@kvark can you review my solution to this?

Reply:
Moved to #1689 (still in draft)